packages/esm-patient-allergies-app/src/allergies/allergy-intolerance.resource.ts

```diff
@@ -2,30 +2,14 @@ import useSWR from 'swr';
 import { map } from 'rxjs/operators';
 import capitalize from 'lodash-es/capitalize';
 import { fhirBaseUrl, openmrsFetch, openmrsObservableFetch, restBaseUrl } from '@openmrs/esm-framework';
-import { type FHIRAllergy, type FHIRAllergyResponse, type ReactionSeverity } from '../types';
-
-export type Allergy = {
-  id: string;
-  clinicalStatus: string;
-  criticality: string;
-  display: string;
-  recordedDate: string;
-  recordedBy: string;
-  recorderType: string;
-  note: string;
-  reactionToSubstance: string;
-  reactionManifestations: Array<string>;
-  reactionSeverity: ReactionSeverity;
-  lastUpdated: string;
-};
-
-type UseAllergies = {
-  allergies: Array<Allergy>;
-  error: Error | null;
-  isLoading: boolean;
-  isValidating: boolean;
-  mutate: () => void;
-};
+import {
+  type FHIRAllergy,
+  type FHIRAllergyResponse,
+  type Allergy,
+  type UseAllergies,
+  type PatientAllergyPayload,
+  type OpenMRSResource,
```

Comment:
`@openmrs/esm-framework` exports an [OpenmrsResource](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-api/src/types/openmrs-resource.ts#L3) interface. Can we use that instead in place of the similarly named annotation in `index.ts`? Also, can we update all references to use the new annotation from the framework?